pkg/controller/cincinnati/names.go

```diff
@@ -18,6 +18,8 @@ const (
 	NameTrustedCAVolume = "trusted-ca"
 	// NameCertConfigMapKey is the ConfigMap key name where the operator expects the external registry CA Cert
 	NameCertConfigMapKey = "cincinnati-registry"
+	// NamePullSecret is the OpenShift pull secret name
+	NamePullSecret = "pull-secret"
```

Comment:
Since I don't think you're expecting this to get used outside of the `namePullSecretCopy` function, is there value in exporting it or even having it as a constant?

Reply:
I'm being a 🐑 here and exporting like every other constant.  Tbh, I think your question can be asked of all these vars.  But, it might be better to deal with this in a follow up PR.

Reply:
I'll take a stab at answering that now. :) None of the other exported constants here are used as part of a resource name. They are all identifiers used within a resource, such as the name of a container of volume. And if you look at the functions below that are generating names, none of them are using exported constants.

The container names are useful outside this package, and probably the volume name is too. For example, they are what someone would use on the command line to run `oc logs -c policy-engine $PODNAME`. They don't change from one operand instance to another. That's why I think they're useful to export, which implicitly documents them.

How resource names are generated is so far not part of this operator's public API, whether the golang API or k8s API. So I think it would be best to not export this.